skt/runner.py

```diff
@@ -306,6 +313,12 @@ def __blacklist_hreq(self, host_requires):
     def __recipe_set_to_job(self, recipe_set, samehost=False):
         tmp = recipe_set.copy()
 
+        try:
+            group = self.get_recipset_group('RS:{}'.format(recipe_set.
+                                                           attrib['id']))
+        except KeyError:
+            group = 'cki'
```

Comment:
If the submitted job has no group then this will add 'cki' to it again, failing if the person who submitted the jobs is not in that group. At least when I check job results for a job without a group there is no group=none attribute

Reply:
Also fixed.